.changeset/twelve-eyes-teach.md

```diff
@@ -2,4 +2,4 @@
 "claude-dev": patch
 ---
 
-Add OpenAI o3 & 4o-mini
+adding o4-mini and o3 models
```

Comment:
Per discussion here: https://github.com/cline/cline/pull/2929#issuecomment-2810521715, may I suggest that we reverse these changes?

Reply:
Done